tools/loader/Loader.cpp

```diff
@@ -31,6 +31,14 @@
 
 using namespace glow;
 
+/// -enable-rowwise : Command line option to enable rowwise quantized
+/// fullyconnected in quantization producure.
+bool enableRowwiseOpt;
+static llvm::cl::opt<bool, true>
```

Comment:
have you run loader with this option succesfuly?

Reply:
Yes.  The test is as follows. Also,  in that test, I added a printf to print out "now in rowwise" once a conversion is successful. There is only one `FC` in resnet50, so there is one "now in rowwise" printed out. 
```
wangm-mbp:build_Release1 wangm$ ./bin/image-classifier tests/images/imagenet/*.png -image-mode=0to1 -m=resnet50 -model-input-name=gpu_0/data -load-profile="profile.yaml" -enable-rowwise
now in rowwise
Model: resnet50
 File: tests/images/imagenet/cat_285.png	Label-K1: 285 (probability: 0.6107)
 File: tests/images/imagenet/dog_207.png	Label-K1: 207 (probability: 0.9519)
 File: tests/images/imagenet/zebra_340.png	Label-K1: 340 (probability: 0.9877)
```